README.md

```diff
@@ -61,12 +61,22 @@ Launches Storybook Test Runner, an isolated development environment for testing
 Require uvicorn inorder to run FastAPI backend server.
 find out more here at https://www.uvicorn.org/.
 
+## `cd backend/`
+It is important to cd into backend folder before running the next step.
+
+## `python -m venv venv`
+
+## `source venv/bin/activate`
+
+## `pip install -r requirement.txt`
+
+## `pip freeze > requirements.txt`
+
```

Comment:
Make sure you update and notate any set-up and set-up instructions

Reply:
will be updated on next push 
can you elaborate on Graphql port and Dev development?

Reply:
Add the localhost with port needed to visit GrapiQL.

Reply:
Add deactivate too for venv. If someone wanna exit

Reply:
updated on d55f7b42f7cff17fde66ffdc9abc25be6f1721ae